docs/pages/kubernetes-access/helm/reference.mdx

```diff
@@ -73,9 +73,11 @@ This reference details available values for the `teleport-cluster` chart.
 
 | Type | Default value | Required? | `teleport.yaml` equivalent | Can be used in `custom` mode? |
 | - | - | - | - | - |
-| `string` | `on` | Yes | `auth_service.authentication.second_factor` | ❌ |
+| `string` | `otp` | Yes | `auth_service.authentication.second_factor` | ❌ |
 
-`authenticationSecondFactor.secondFactor` controls the second factor used for local user authentication. Possible values are `on`, `optional` and `webauthn`.
+`authenticationSecondFactor.secondFactor` controls the second factor used for local user authentication. Possible values are `off` (not recommended), `on`, `otp`, `optional` and `webauthn`.
+
+When set to `on` or `webauthn`, the `authenticationSecondFactor.webauthn` section must also be set.
```

Comment:
This more accurately represents the functionality.
```suggestion
When set to `on` or `optional`, the secondFactor used will default to `webauthn` or `u2f`, depending on whether `authenticationSecondFactor.webauthn` or `authenticationSeconfFactor.u2f` is set.
```
If the issue is that you set `on` and set the `u2f` field and still got the error log, then this must be a bug.

Reply:
For v8 and v9, yes. But on master we've dropped u2f support.

Reply:
@Joerger is correct, we are missing "optional" in the list. It functions rather similarly to "on" in terms of configuration.

I don't see an U2F section in the schema, did we ever support it in the chart? U2F settings are relevant for backwards compatibility.

It also looks like we are setting the cluster name as the WebAuthn RPID? In this case, shouldn't the webauthn section always be present?

Reply:
Got it. Given that I'm only intending to backport this to v8 and v9 (which both have decent webauthn support) and there's no support for `authenticationSecondFactor.u2f` in the Helm chart, I don't think I'll mention `u2f` here at all for ease.

I do think I should add `optional` to the list of values which require the `webauthn` section to be set, though.

Reply:
Also, I understand your comments about `u2f` but I'm loath to introduce another full section and associated Helm parsing/validation for a standard which I think we now consider deprecated in favour of webauthn. Any existing Helm chart users who wanted U2F are likely to be deploying a custom Teleport config which enables it, so I think we can leave that as-is.

Reply:
> we now always write rp_id when second factor is not set to ...